Batch Insight crash if instrumentation key is missing

Hi, any ideas why it does not work on Ubuntu 16.04?

Here is a log:
```
2019-03-15 10:41:49 (12.1 MB/s) - './batch-insights' saved [7474080/7474080]

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x6719b9]

goroutine 1 [running]:
github.com/Azure/batch-insights/pkg.hideSecret(...)
	/home/vsts/work/1/s/pkg/config.go:148
github.com/Azure/batch-insights/pkg.UserConfig.Print(0xc000010d40, 0xc000010d50, 0x0, 0xc000010da0, 0x1, 0x1, 0xc000018578, 0xc000010db0, 0x1, 0x1)
	/home/vsts/work/1/s/pkg/config.go:32 +0x159
main.main()
	/home/vsts/work/1/s/main.go:94 +0x43e
debconf: unable to initialize frontend: Dialog
debconf: (Dialog frontend will not work on a dumb terminal, an emacs shell buffer, or without a controlling terminal.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (This frontend requires a controlling tty.)
debconf: falling back to frontend: Teletype
dpkg-preconfigure: unable to re-open stdin: 
```

--- a/pkg/config.go
+++ b/pkg/config.go
@@ -29,7 +29,9 @@ func (config UserConfig) Print() {
 	fmt.Printf("User configuration:\n")
 	fmt.Printf("   Pool ID: %s\n", *config.PoolID)
 	fmt.Printf("   Node ID: %s\n", *config.NodeID)
-	fmt.Printf("   Instrumentation Key: %s\n", hideSecret(*config.InstrumentationKey))
+	if config.InstrumentationKey != nil {
+		fmt.Printf("   Instrumentation Key: %s\n", hideSecret(*config.InstrumentationKey))
+	}
 	fmt.Printf("   Aggregation: %d\n", *config.Aggregation)
 	fmt.Printf("   Disable: %v\n", config.Disable)
 	fmt.Printf("   Monitoring processes: %v\n", config.Processes)

